lib/vendor/noitidart-ostypes/cutils.jsm

```diff
@@ -0,0 +1,462 @@
+/*** CTypes Utilities ***
+ * Stuff that I find useful in my CTypes work under all Operating Systems
```

Comment:
> We should be able to do this now with npm i Noitidart/ostypes -D (until it gets published on npm, at which point we can just do npm i ostypes -D). But we'd still need an npm postinstall hook to copy the .jsm files from ./node_modules/ostypes/*.jsm to ./lib/vendor/noitidart-ostypes/*.

@pdehaan That's a great idea! Would you be interested in opening a PR against this branch?